### PR TITLE
Rebrand balances

### PR DESCRIPTION
**Pull request by etharner**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
Tested in FF, Safari, Brave, mob browser for Android, web wallet + FF/Chrome ext:
Change tokens/currency - pass 
Send ae - pass 
Receive ae - pass 
Send FT - pass 
Receive FT - pass 
Balances view- pass 
Send via scan QR - pass 
AE/FT are displayed in Tx History (All/Sent/Received) - pass 
Send Tip - pass 
Claim tip - pass 
Tx history - pass 
Tx details - pass 
Invite link: create, top up, claim back - pass 
Register a name - pass ( in the testnet) 
Set name pointer - pass 
Set as default name - pass 
Extend - pass 
Settings - pass 
Networks - pass 

Error with AE tokens in Balances tab: Send, Receive, Tip (FF web and extension):
balances --payments
cons1

https://user-images.githubusercontent.com/69896204/124292251-0a3b4f80-db5e-11eb-9cfa-ea99ff5892cc.mov

The same error in Chrome extension and Web wallet in Brave/Chrome browser:
In chrome extension
chr

and the same in Web wallet in Safari:
safari
saf

The withdrawal transaction is displayed in the Received field:
Tx hash: th_2gQgxFPHAxnQHoJ2zikavzkEpBmidhF7c9WzPUrmNdmMA4zayv
min amount

Several transactions in the wallet are dated incorrectly:
date

An invite link is created in FT, although the amount is displayed in AE tokens:
invite with FT

File: src/popup/locales/en.json [hunk 1]
```diff
@@ -457,13 +456,13 @@
...
       "tokens": "Tokens",
-      "searchPlaceholder": "Search by token name or community...",
+      "searchPlaceholder": "Search for token",
       "all": "All",
       "myTokens": "My Tokens",
       "mcap": "MCap",
       "price": "Price",
       "not-available": "N/A",
-      "no-results": "There are no tokens to display"
+      "no-results": "No tokens in the account balance"
```
Comment (kenodressel) [inline src/popup/locales/en.json:465]:
```suggestion
      "no-results": "This account has no tokens with balance"
```

**Review by kenodressel — APPROVED** [review]

File: src/popup/router/components/TokenAmount.vue [hunk 2]
```diff
@@ -27,17 +34,16 @@ export default {
...
   },
   computed: {
     amountRounded() {
       return +this.amount.toFixed(this.amount < 0.01 ? 9 : 2);
     },
     ...mapState({
       amountFiat(state, { convertToCurrency, formatCurrency }) {
-        if (this.symbol !== 'AE') return false;
-        const converted = convertToCurrency(this.amount);
-        if (converted < 0.01 || this.hideFiat) return false;
```
Comment (imollov) [inline src/popup/router/components/TokenAmount.vue:39]: I recall @onvisions asked to not show zero fiat conversion when implementing this. Is it synced with him or the design team?

Comment (etharner) [reply src/popup/router/components/TokenAmount.vue:39]: That's how it looks in designs

Comment (imollov) [reply src/popup/router/components/TokenAmount.vue:39]: Which designs? Maybe is a different use-case and we need to split this component.

**Review by imollov — APPROVED** [review]

**Pull request merged**